docs/index.md

```diff
@@ -32,3 +32,8 @@
 
    + [Custom Devices Tips & Tricks](Custom_Device_Tips_and_Tricks.md)
    + [Scripting APIs for Custom Devices](Scripting_API.md)
+
+#### Migrating from an LLB to a PPL based Custom Device
```

Comment:
Please remove this section, and we will add the new page under "Custom Device Tips and Tricks"